dashboard-for-desktop/articles/dashboard-designer/data-shaping/filtering.md

```diff
@@ -4,31 +4,79 @@ author: Natalia Kazakova
 legacyId: 16532
 ---
 # Filtering
-The **Dashboard** allows you to filter a [query](../working-with-data/filter-queries.md) of the [SQL Data Source](../providing-data/connecting-to-sql-databases.md) or apply filtering to a specific data-aware dashboard item.
+The Dashboard Designer allows you to filter data in the dashboard items or apply filters to a specific measure. [Dimensions](../binding-dashboard-items-to-data/binding-dashboard-items-to-data.md) and [hidden dimensions](../binding-dashboard-items-to-data/hidden-data-items.md) are used to build filter criteria.
 
-This topic describes how to enable and reset filtering.
+## Dashboard Item Filter
 
-## Apply Filtering
-To configure filtering, select the target dashboard item and do one of the following.
-* If you are using a Ribbon menu, click the **Edit Filter** button in the **Data** tab.
+Filters that apply to a [dashboard item](../designing-dashboard-items.md) affect only this item. To add filters, select the target dashboard item and do one of the following:
+
+* Click the **Edit Filter** button in the **Data** tab in a Ribbon menu. 
```

Comment:
To avoid saying "in" twice, I'd like to change the second one to "of":

```suggestion
* Click the **Edit Filter** button in the **Data** tab of a Ribbon menu. 
```